src/cabinetry/model_utils.py

```diff
@@ -300,16 +294,23 @@ def yield_stdev(
         up_comb = np.vstack((up_comb, np.sum(up_comb, axis=0)))
         # turn into list of channels (keep all samples, select correct bins per channel)
         # indices: channel, sample, bin
-        up_yields = [
+        up_yields_per_channel = [
             up_comb[:, model.config.channel_slices[ch]] for ch in model.config.channels
         ]
-        # append list of yields summed per channel
-        up_yields += [
-            np.asarray(np.sum(chan_yields, axis=-1, keepdims=True))
-            for chan_yields in up_yields
-        ]
-        # indices: variation, channel, sample, bin
-        up_variations.append(up_yields)
+        # calculate list of yields summed per channel
+        up_yields_sum = np.stack(
+            [
+                np.sum(chan_yields, axis=-1, keepdims=True)
+                for chan_yields in up_yields_per_channel
+            ]
+        )
+        # reshape to get rid of extra axis
```

Comment:
```suggestion
        # reshape to drop bin axis, transpose to turn channel axis into new bin axis
        # (channel, sample, bin) -> (sample, bin) where "bin" becomes channel sums
```
The transpose here was confusing to me at first and I'm sure I'll forget again in the future what it does so I'd like to make the comment very verbose.